portal/src/main/java/org/bonitasoft/console/client/admin/process/view/ProcessListingAdminPage.java

```diff
@@ -114,8 +114,8 @@ private ItemListingFilter newUnresolvedProcessesFilter() {
     protected ItemListingResourceFilter defineResourceFilters() {
         final ItemListingResourceFilter categories = new ItemListingResourceFilter(
                 new APISearchRequest(Definitions.get(CategoryDefinition.TOKEN)),
-                CategoryItem.ATTRIBUTE_NAME, "default-image" /* fake attribute to display default image */, TABLE_NO_ACTION)
-                .addFilterMapping(ProcessItem.FILTER_CATEGORY_ID, CategoryItem.ATTRIBUTE_ID);
+                CategoryItem.ATTRIBUTE_NAME, TABLE_NO_ACTION)
```

Comment:
It follows the code style from https://github.com/bonitasoft/bonita-internal-tools/tree/master/codestyle

It is the same on bonita-web